webpack.config.js

```diff
@@ -0,0 +1,212 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
```

Comment:
This license version does not look correct. Did you confirm that this is the latest license that we are using?